src/api/public/apidocs-new/paths/source_project_cmd_createkey.yaml

```diff
@@ -0,0 +1,45 @@
+post:
+  summary: Generate a new gpg key for a project
+  description: Generate a new gpg key. If the project already has a gpg key, the old key is discarded.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      description: ok
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            type: object
+            properties:
+              code:
+                type: string
+                xml:
+                  attribute: true
+            xml:
+              name: status
+          example:
+            code: ok
```

Comment:
What about using `$ref: '../components/responses/succeeded.yaml'`? :thought_balloon:

Reply:
`succeeded.yaml` is different. It has both the `summary and `code`. In this endpoint, we only get `code` in response.